composer.json

```diff
@@ -0,0 +1,13 @@
+{
+    "name": "alleyinteractive/wp-block-migrator",
+    "description": "Utilities for assisting with Gutenberg migrations.",
+    "type": "wordpress-plugin",
+    "license": "GPLv2",
+    "authors": [
+        {
+            "name": "Alley Interactive",
```

Comment:
Worth a double check about whether this should be just "Alley" -- Although the github org name remains Alley Interactive, we've dropped "Interactive" from the company name for business and practical purposes.

Reply:
Good call. Some repos have it, some don't. Going off of `wp-bulk-task`, I'll remove it, since it looks like it isn't there in the more recent ones.